config/runtime.exs

```diff
@@ -17,3 +17,22 @@ config :ueberauth, Ueberauth.Strategy.Cognito,
   client_secret: System.get_env("COGNITO_CLIENT_SECRET"),
   user_pool_id: System.get_env("COGNITO_USER_POOL_ID"),
   aws_region: System.get_env("COGNITO_AWS_REGION")
+
+if config_env() == :dev do
+  config(:document_viewer, live_catalog?: false)
+
+  config(:document_viewer, DocumentViewerWeb.Endpoint,
+    http: [port: 4000],
+    secret_key_base: "GQz2AbfSliQp6FYTncpcZJgM7skJdhKH6refgghsdE9gOz0TQ5u5+Er+tzH6XmUt",
+    debug_errors: true,
+    live_catalog?: false,
+    check_origin: false
+  )
+end
+
+if config_env() == :test do
```

Comment:
question: is there a reason this shouldn't live in `config/test.exs`?

Reply:
No, but on line 7 above we set the secret_key_base to be runtime config so locally and in tests I was getting an error. I assume we want secret_key_base to be read at runtime for prod only, so I could instead put `if config_env() == :prod do` above instead of this. Maybe that's better? But I kind of feel like maybe it's better for it to all be runtime or all be compile time.

Reply:
None of that configuration is runtime configuration though, as far as I can see? It's all static, so known at compile-time?

Reply:
secret key base is https://github.com/mbta/document_viewer/pull/403/files#diff-e43c5cbf91db7a8062b6cb860cbf118296c1b4c7ee32fdcf702e54234ba38092R8  so if I don't do this the app starts, looks for an env var that isn't there.

Reply:
Gating that on `:prod` makes sense: see https://github.com/mbta/my_charlie/blob/main/config/runtime.exs#L186-L205 for how MyCharlie does it.